recipe/meta.yaml

```diff
@@ -354,51 +320,35 @@ outputs:
         - {{ cdt('xcb-util-wm-devel') }}          # [linux and not (aarch64 or x86_64)]
         - jom                                    # [win]
         - make  # [not win]
-        - python
-        - pyqt-builder
-        - dbus-python  # [unix]
         - pkg-config
-        - pyqt
-        - qt-main 5.15.*
-        - qt-webengine 5.15.*
-        - qtwebkit 5.*
-        - setuptools
-        - wheel
-        - pip
       host:
         - python
-        - pip
-        - setuptools
-        - wheel
-        - sip
-        - toml
-        - pyqt-builder
-        - packaging
+        - pyqt-builder >=1.9,<2
+        - sip >=5.3,<7
         - qt-main 5.15.*
-        - qt-webengine 5.15.*
-        - qtwebkit 5.*
-        - pyqt
+        - {{ pin_subpackage('pyqt', exact=True) }}
         - {{ cdt('mesa-libgl-devel') }}          # [linux]
         - {{ cdt('mesa-libegl-devel') }}         # [linux]
       run:
         - qt-main 5.15.*
-        - qtwebkit 5.*
-        - qt-webengine 5.15.*
         - python
         - {{ pin_subpackage('pyqt', max_pin='x.x') }}
-{% if enable_testingui %}
     test:
+{% if enable_testingui %}
       imports:
         - PyQt5.QtChart
 {% endif %}
+      requires:
+        - pip
+      commands:
+        - pip check
 
 about:
   home: https://www.riverbankcomputing.com/software/pyqt/
   license: GPL-3.0-only
   license_family: GPL
   license_file: pyqt/LICENSE
```

Comment:
The README file says:
```

License
-------

PyQt5 is released under the GPL v3 license and under a commercial license that
allows for the development of proprietary applications.
```

Reply:
I am of the opinion of leaving it as is. The commercial license is not included in the sdist. The only reference is this:
```
# If you do not wish to use this file under the terms of the GPL version 3.0
# then you may purchase a commercial license.  For more information contact
# info@riverbankcomputing.com
```

Reply:
It is GPL 3. Like Charles quoted, you only need a commercial license if you don't want to release your softwre with a GPL 3 or LGPL compatible license. See https://riverbankcomputing.com/commercial/license-faq.